cmd/tailscaled/depaware.txt

```diff
@@ -170,6 +176,9 @@ tailscale.com/cmd/tailscaled dependencies: (generated by github.com/tailscale/de
         golang.org/x/crypto/salsa20/salsa                            from golang.org/x/crypto/nacl/box+
         golang.org/x/net/bpf                                         from github.com/mdlayher/netlink+
         golang.org/x/net/dns/dnsmessage                              from net+
+        golang.org/x/net/html                                        from golang.org/x/net/html/charset
+        golang.org/x/net/html/atom                                   from golang.org/x/net/html
+        golang.org/x/net/html/charset                                from github.com/huin/goupnp
```

Comment:
We're very very memory constrained on iOS. I don't think we want to be pulling in large Unicode tables on iOS.

This dependency edge seems like a problem. Is it needed? Do we have to fork the goupnp package?

Reply:
I've forked the package, and removed the dependency on the charset. There is still more in the package, can work on trimming it down to something more reasonable.